perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/MembersManagerBlImpl.java

```diff
@@ -138,15 +138,6 @@ public void deleteMember(PerunSession sess, Member member) throws MemberAlreadyR
 			throw new ConsistencyErrorException("Removing member who doesn't have corresponding user.", e1);
 		}
 
-		List<Facility> allowedFacilities = getPerunBl().getFacilitiesManagerBl().getAllowedFacilities(sess, user);
```

Comment:
It seems that the user object is no longer needed, so you can remove also its initialization

Reply:
You are right, I have removed it.